### ISSUE TITLE
Examples not working

### ISSUE THREAD

**Issue by Nutomic** [dev-sec/cis-docker-benchmark#58]
I tried the following commands, but both of them only result in an error `Input 'container_capadd' is required and does not have a value.`. The readme says I need a yml attribute file, but there is no mention how this file can be passed to the script.

```
# run profile locally
$ git clone https://github.com/dev-sec/cis-docker-benchmark
$ inspec exec cis-docker-benchmark

# run profile locally and directly from Github
$ inspec exec https://github.com/dev-sec/cis-docker-benchmark
```

**OS / Environment**
Ubuntu 18.04.2 LTS

**Inspec Version**

```
3.7.1
```

**Baseline Version**

```
a281d230fbe52f9ef7f88c5a4147dbce6af07ae9
```

**Comment by chris-rock** [comment]
Hi @Nutomic thank you for the feedback. The project includes a sample attributes file and you can execute it via:

```
# run profile on remote host via SSH with sudo and define attribute value
inspec exec cis-docker-benchmark --attrs sample_attributes.yml
```
It is mentioned in the [readme](https://github.com/dev-sec/cis-docker-benchmark#usage) but maybe not prominent enough.

**Comment by Nutomic** [comment]
Right, I only looked at the first examples for running locally, and there is not mention how I can pass the attributes. It worked now, but I still had to edit the sample file. Would be good if it worked out of the box.

**Comment by chris-rock** [comment]
@Nutomic would you like to open a PR to improve it?

**Comment by Nutomic** [comment]
Sorry but I dont have the time.

**Comment by chris-rock** [comment]
Would it be possible to share your updated attributes within this issue?

**Comment by chris-rock** [comment]
fixed in #61
